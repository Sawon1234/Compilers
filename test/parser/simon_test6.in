class Fib (v : Int) extends InterviewQuestions {
    super();

    fun fib(n : Int) : Int {
        if ( !n ) {
            return 1; 
        }
    }
}

return 5;