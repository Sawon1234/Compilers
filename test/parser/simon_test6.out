class Fib < > ( v : Int < > ) extends InterviewQuestions < > { super ( ) ; fun fib < > ( n : Int < > ) : Int < > if ( n . negate < > ( ) ) return 1 ; else { } } return 5 ;